### DISCUSSION TITLE
load an image ignoring EXIF

### DISCUSSION THREAD

**Discussion by mgarato** [SixLabors/ImageSharp#1701] [Q&A] [answered]
hi,
i need a way to load an image ignoring the EXIF,
i'm currently doing something like:
using var image = Image.Load(imageBytes);

**Comment by JimBobSquarePants** [comment, marked as answer]
If you know the image format you can pass an explicit decoder implementation setting the `IgnoreImageMetadata` property to true.

If not, one loaded set the Metadata.Exif property to `null`.

https://docs.sixlabors.com/api/ImageSharp/SixLabors.ImageSharp.html
